jwst/exp_to_source/tests/helpers.py

```diff
@@ -1,17 +1,15 @@
 # Testing helpers
-from glob import glob
-import os
+from pathlib import Path
 
 
-INPUT_FILES_GLOB = 'data/jwst_nod?_cal.fits'
+INPUT_FILES_GLOB = "data/"
 
 
 def t_path(partial_path):
-    """Construction the full path for test files"""
-    test_dir = os.path.dirname(__file__)
-    return os.path.join(test_dir, partial_path)
+    """Construct the full path for test files."""
+    test_dir = Path(__file__).parent
+    return Path(test_dir) / partial_path
 
 
 # Calculate some extra constants
-INPUT_FILES_GLOB = t_path(INPUT_FILES_GLOB)
-INPUT_FILES = glob(t_path(INPUT_FILES_GLOB))
+INPUT_FILES = list(t_path("data").glob("jwst_nod*_cal.fits"))
```

Comment:
Is there any reason this can no longer be `jwst_nod?_cal.fits`?  It looks like the original intent was only to have a single-character wildcard, not a multi-character one.